src/sidebar/components/CircularProgress.tsx

```diff
@@ -0,0 +1,61 @@
+export type CircularProgressProps = {
+  /** Width and height of the indicator in pixels. */
+  size: number;
+
+  /** Progress value between 0 and 100. */
+  value: number;
+};
+
+/**
+ * A compact circular progress indicator.
+ */
+export default function CircularProgress({
+  size,
+  value,
+}: CircularProgressProps) {
+  const strokeWidth = 2;
+
+  // Internal diameter of circle.
+  const diameter = size - 2 * strokeWidth;
+  const circumference = Math.PI * diameter;
+
+  return (
+    <span
+      // This assumes a dark background. We'll need a variant for light
+      // backgrounds at some point.
+      className="text-grey-3"
+      role="progressbar"
+      aria-valuenow={value}
```

Comment:
We specify the _current_ value here, but not the maximum because the default is 100. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role.